### ISSUE TITLE
Remove the minor crimes / major crimes fields from security records.

### ISSUE THREAD

**Issue by despotate** [d3athrow/vgstation13#14531]
These just add confusion rather than removing it. Better to simplify it down to only the "add/view comments" field. There's no real consistency or intuitiveness to the minor/major crime fields. They also don't have a chronological order to them or commenters' name attached, unlike the comments field.

With just the comments field, sec would intuitively add comments like this:

- reported by x for crime y [officer's name]
- confirmed committing crime y [officer's name]
- brigged for z minutes for crime y [officer's name]
- paroled with implant a [officer's name]
- confirmed stealing the captain's spare and murdering fifteen people, kill on sight [suspicious officer's name]
- disregard the above comment, that was written by [not an officer] who stole a secHUD [officer's name]
- committed crime y again [officer's name]
- brigged for b minutes for crime y [officer's name]
- committed crime c [officer's name]
- put in perma for crime c [officer's name]

Now try imagining the above comments but spread out across the minor/major crime fields and their subcomments, it'd be a hellhole, eg. the same crime y having to be put into the same field twice, no chronological order, no idea if the guy'd already served his sentence yet, the dude'd have gotten shot by officers because names aren't logged in those fields, etc.

theoretically the minor/major crimes have a purpose in that they're the "at a glance" version of the comments but sec really has enough complexity to deal with as is, and most of the time word of mouth is used over sec paperwork anyway, so it's a much better idea to simplify the HUD paperwork.

**Comment by Fruktlimpa** [comment]
Can't we have so that upon inspecting someone with a comment it will display the most recent one instead of having to fiddle with the menu? And if you want to read all the comments you open the menu and read it. This probably isn't possible as I don't see a way you could tell the game to only access the most recent line instead of the whole shit.

**Comment by eplgr** [comment]
I was gonna suggest make "comments" function like "minor/major crimes" and remove the latter, but I like @Fruktlimpa 's idea way more.
It could also be renamed to "Crimes & comments".

Or instead, we could do so something like:
if you write `/important [something]`, it shows up in the same way minor/major crimes show up.

Also, you never know whether someone does have comments or doesn't, so it'd be nice that if they had, it would say so.
Like, near the button: **[ADD COMMENT]** _This person has # comments written about him._

**Comment by despotate** [comment]
Good QOL ideas.

>  if you write /important [something], it shows up in the same way minor/major crimes show up.

Sec doesn't have time for this and almost no one would be aware of this feature. While the additions suggested here are nice, we need to chill and just get it down to the basics for now. As it stands 99% of the time people just `:s What did this guy do` because of how unreliable and unused the comments are.

edit: if there was a QOL change _I'd_ suggest it's that hitting enter on the "add comments" field would automatically submit the comment for you instead of adding a newline. But I'm pretty sure this is a BYOND limitation.

I play sec a lot and it's hell even adding a simple comment because of how many steps it takes and how pointless it ends up being in the end.

**Comment by eplgr** [comment]
But it'd be super easy and simple. Maybe I did not express myself very well:

>Basically, we would remove everything and only have one button: [ADD CRIMES & COMMENTS]
Whatever you do, it will add one line in the user's "crimes & comments" (which is just a renamed version of comments).
Now, if you add `/important` at the start, it will *also* show up when you examine that person. We could illustrate this, with big red letters.
Just *one* extra word doesn't sound like a problem to me.
Oh, and also, the button [ADD CRIMES & COMMENTS] next to it would also state whether the user does have notes, or doesn't.
Like I said before, it'd be:
**[ADD CRIMES & COMMENTS]** _This person has # comments written about him._

**tl;dr version:** remove everything, keep "comments", and if you want a comment to show up when you examine someone, add `/important` at the start.

**Comment by Aurx** [comment]
Ideally, setting somebody to arrest would prompt you for a reason, that would then show up when examining them through a sechud.

**Comment by Kurfursten** [comment]
Back in 2015 @nuklearcellphoneg had an idea for revamping the cell timer to basically have record-keeping built in. It's also good to tag him just because he always want to give input on sec stuff.

#6942

**Comment by nuklearcellphoneg** [comment]
If something like #6942 was added in then it would be a lot easier to add the crimes at the cell door and add them to their criminal history.

So you could swipe the inmates ID(presumably part of stripping them) which brings up their record, and from there have the space law crimes listed out as buttons, which would add the corresponding time to the timer, and it could also simultaneously add these crimes to the inmates record. Optimally it would also recognize repeat offenses and tack on the extra time.

It would take some of the work out of record keeping and remove some of the guess work out of brig times and criminal history.
